### DISCUSSION TITLE
Supervised HAOS

### DISCUSSION THREAD

**Discussion by FordMustang0288** [tteck/Proxmox#228] [Q&A] [answered]
@tteck - The proxmox update went well. Upon trying to use the HA LXC, I realized it would not work for me as I need a supervised mode with add-ons. I then tried the VM route but that appears to install CORE which again doesn't have supervisor.

Any chance you can point me to the next best option? I love your scripts and I'm wondering if I need to go back to the whisker github page or if you have another suggestion.

Thanks

EDIT: I should mention that I am trying to run frigate via home assistant

**Comment by tteck** [comment, marked as answer]
The Home Assistant OS VM script installs the OS version which contains the supervisor. https://github.com/tteck/Proxmox/discussions/54#discussioncomment-2410978
The whiskerz007 script no longer works. It downloads the wrong image.

**Comment by FordMustang0288** [comment]
Good morning, thank you for the reply.

I'll have to take a look later to see exactly where I thought it said core for the version.

If it installs OS, I should have a supervisor option in the sidebar, correct? I do not, unless there is something I need to do to enable it.

I know these are rookie questions, so I apologize. Even being a little familiar with this stuff, I still find it difficult to track what is what.

**Reply by tteck** [reply]
https://www.home-assistant.io/blog/2021/12/11/release-202112/#brand-new-configuration-panel

**Comment by FordMustang0288** [comment]
Looks like it is very possible I overlooked the add ons option in the settings. I wish I could take a look now, but I appreciate the hand holding!!
